frontend/src/metabase/admin/databases/containers/DatabaseEditApp.jsx

```diff
@@ -168,12 +179,16 @@ class DatabaseEditApp extends Component {
                                   onChangeField("engine", engine)
                                 }
                               />
+                              <FormError
+                                className="mt3 mb4"
+                                anchorMarginTop={24}
```

Comment:
`anchorMarginTop` is used to determine the anchor offset when scrolling back to the error message. Otherwise, the top of the element will flush with the navbar which doesn't look too good.

p.s. `mt3` = `24px`

### Admin > Databases

#### with `anchorMarginTop`
![image](https://user-images.githubusercontent.com/1937582/169346136-f607d617-2d5c-4b68-b025-683240ec8e9b.png)

#### Without `anchorMarginTop`
![image](https://user-images.githubusercontent.com/1937582/169346336-4984278b-75eb-4212-98c8-dc90aecbf1f2.png)